added new tag and value to exempt from locking RG (#48)
hmcts/azure-resource-locks: README.md

- [Disable-resource-locks](https://dev.azure.com/hmcts/PlatformOperations/_build?definitionId=535)
    - Select the subscription and resource group(s) to run against from the job parameters

## Exempt from autolocking

> To avoid your resources from being auto locked, you can use the `exemptFromAutoLock` tag  on the resource group:

Reviewer: ```suggestion
To avoid your resources from being auto locked, you can use the `exemptFromAutoLock` tag on the resource group:
```

> - In order to exempt from auto locking resource, you can use below tag on the resource group which will filter out the resource group from being auto locked by Enable-resource-locks pipeline.

Reviewer: ```suggestion
To avoid your resources from being auto locked, you can use the `exemptFromAutoLock` tag  on the resource group:
```
